client/my-sites/plan-features/index.jsx

```diff
@@ -679,6 +679,14 @@ export class PlanFeatures extends Component {
 		const description = feature.getDescription
 			? feature.getDescription( abtest, this.props.domainName )
 			: null;
+
+		const recordEvent = feature.getTrackingLabel
```

Comment:
I'm not particularly happy with how we declare a handler function on every re-render. This should be defined as a component method instead.